src/pepr/tasks.yaml

```diff
@@ -28,17 +33,29 @@ tasks:
             exit 1
           fi
           $HOME/go/bin/addlicense -l "AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial" -s=only -v -c "Defense Unicorns" src/pepr/operator/crd/generated
+          $HOME/go/bin/addlicense -l "AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial" -s=only -v -c "Defense Unicorns" src/pepr/uds-cluster-crds/templates
 
       - task: gen-docs
 
       - cmd: "npx pepr format"
 
+  - name: gen-manifests
+    description: "Generate CRD manifests"
+    actions:
+      - cmd: |
+          CRD_LIST="exemptions.uds.dev packages.uds.dev clusterconfig.uds.dev"
+          for CRD in $CRD_LIST; do
+             FILE="src/pepr/uds-cluster-crds/templates/$CRD.yaml"
+             kubectl get crd $CRD -o yaml > $FILE
+             yq -i 'del(.metadata.creationTimestamp,.metadata.generation,.metadata.resourceVersion,.metadata.uid,.status)' $FILE
```

Comment:
These two commands should be `uds zarf tools` just to make sure this works well on dev machines where kubectl or yq aren't present/on the path.